Doc/library/collections.abc.rst

```diff
@@ -98,12 +98,20 @@ ABC                        Inherits from          Abstract Methods        Mixin
 
 
 .. class:: Container
-           Hashable
-           Sized
-           Callable
 
-   ABCs for classes that provide respectively the methods :meth:`__contains__`,
-   :meth:`__hash__`, :meth:`__len__`, and :meth:`__call__`.
+   ABC for classes that provide method :meth:`__contains__`.
+
+.. class:: Hashable
+
+   ABC for classes that provide method :meth:`__hash__`.
+
+.. class:: Sized
+
+   ABC for classes that provide method :meth:`__len__`.
+
+.. class:: Callable
+
+   ABC for classes that provide method :meth:`__call__`.
 
```

Comment:
Actually I just noticed - the wording is inconsistent with the other items, like, just below it says:

"ABC for classes that provide the :meth:`__iter__` method."

and not

"ABC for classes that provide method :meth:`__iter__`."

Reply:
Excellent eye. I've updated it to be consistent.